unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderEntity/DCLBuilderInWorldEntity.cs

```diff
@@ -205,30 +205,17 @@ public void Delete()
         OnDelete?.Invoke(this);
     }
 
-    public bool HaveSmartItemComponent()
-    {
-        return rootEntity.components.ContainsKey(CLASS_ID_COMPONENT.SMART_ITEM);
-    }
-
-    public void CreateColliders()
-    {
-        if (rootEntity.meshRootGameObject && rootEntity.meshesInfo.renderers.Length > 0)
-        {
-            CreateCollidersForEntity(rootEntity);
-        }
-    }
-
     public void DestroyColliders()
     {   
-        foreach (List<GameObject> entityCollider in collidersDictionary.Values)
+        foreach (List<GameObject> entityCollider in collidersGameObjectDictionary.Values)
```

Comment:
```suggestion
        foreach (List<GameObject> entityGameObject in collidersGameObjectDictionary.Values)
```
Let's clarify this so `Destroy(entityCollider)` doesn't look like a bug (destroying a component wouldn't destroy the GameObject)